### PR TITLE
receivers: attachement indexer

### PR DESCRIPTION
**Pull request by jirikuncar**
* Adds new receiver for automatic fulltext indexing.

Co-authored-by: Salvatore Zaza <salvatore.zaza@gmail.com>
Co-authored-by: Rémi Ducceschi <remi.ducceschi@gmail.com>
Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

--
addresses https://github.com/inveniosoftware/invenio-indexer/issues/46

### CODE REVIEW

File: invenio_records_files/mappings/records-files/records-files-v1.0.0.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "checksum": {
+              "type": "string"
+            },
+            "size": {
+              "type": "integer"
+            },
+            "version_id": {
+              "type": "string"
+            },
+            "file": {
```
Comment (tiborsimko) [inline invenio_records_files/mappings/records-files/records-files-v1.0.0.json]: `_files.file.content` or `_files._attachment.content`?

Comment (jirikuncar) [inline invenio_records_files/mappings/records-files/records-files-v1.0.0.json]: `_files._attachment`

Comment (tiborsimko) [inline invenio_records_files/mappings/records-files/records-files-v1.0.0.json]: :+1:

**Comment by jirikuncar** [comment]
LGTM based on https://github.com/inveniosoftware/invenio-records-files/pull/35#discussion_r84243895

**Pull request merged**